pkg/agent/monitortool/monitor_test.go

```diff
@@ -381,6 +397,83 @@ func TestUpdateMonitorPingInterval(t *testing.T) {
 	assert.EventuallyWithT(t, func(t *assert.CollectT) {
 		packets = collect(packets)
 		assert.ElementsMatch(t, []string{"10.0.2.1", "10.0.3.1"}, extractIPs(packets))
+		assert.Equal(t, 0, int(reportCount.Load()), "Expected no report yet (jitter still pending)")
+	}, 2*time.Second, 10*time.Millisecond)
+
+	// Advance by 1 more second (reportJitter) → total 91s: report should now occur
+	fakeClock.Step(1 * time.Second)
+	assert.EventuallyWithT(t, func(t *assert.CollectT) {
+		assert.Equal(t, 1, int(reportCount.Load()), "Expected report after jittered interval (total 91s)")
+	}, 2*time.Second, 10*time.Millisecond)
+}
+
+func TestPingIntervalBelowMinReportInterval(t *testing.T) {
+	// While investigating test flakiness in CI, we enabled verbose logging.
+	var level klog.Level
+	level.Set("4")
+	defer level.Set("0")
+
+	stopCh := make(chan struct{})
+	defer close(stopCh)
+	// Create a NodeLatencyMonitor with 5s ping interval (below minReportInterval of 10s)
+	nlmBelowMinReportInterval := &crdv1alpha1.NodeLatencyMonitor{
```

Comment:
just name this `nlm`. There is no need for a long variable name within the context of the test function.